_includes/footer.html

```diff
@@ -30,11 +30,11 @@ <h3 class="widget-title uppercase">Download</h3>
           <dl>
             <dd><a href="https://getfedora.org/en/workstation/download/">Get Fedora Workstation</a></dd>
             <dd><a href="https://getfedora.org/en/server/download/">Get Fedora Server</a></dd>
-            <dd><a href="https://alt.fedoraproject.org/en/cloud/">Get Fedora Cloud</a></dd>
+            <dd><a href="https://getfedora.org/en/atomic/download/">Get Fedora Atomic</a></dd>
```

Comment:
Actually, Fedora Atomic and Cloud are two distinct things.
Maybe we should have both?